src/commonMain/kotlin/exchange.dydx.abacus/output/PerpetualState.kt

```diff
@@ -119,4 +120,8 @@ data class PerpetualState(
     fun transferStatus(hash: String): TransferStatus? {
         return transferStatuses?.get(hash)
     }
+
+    fun trackStatus(hash: String): Boolean? {
+        return trackStatuses?.get(hash)
+    }
```

Comment:
is this just a convenience method for map access? Ideally we access maps uniformly throughout the app IMO